weekly_meeting_notes/sept_20.md

```diff
@@ -0,0 +1,84 @@
+# 🚀 Sept 20th 
+**Start Time** 9:30am
+
+**End Time** 10:30am
+
+**Location**
```

Comment:
can add that we had the meeting over Discord

Reply:
Added!